app/src/openxr/cpp/OpenXRInputMappings.h

```diff
@@ -143,7 +149,10 @@ namespace crow {
         },
         std::vector<OpenXRAxis> {
             { OpenXRAxisType::Thumbstick, kPathThumbstick,  OpenXRHandFlags::Both },
-        }
+        },
+        std::vector<OpenXRHaptic> {
+            { kPathHaptic, OpenXRHandFlags::Both },
```

Comment:
This is wrong Oculus Touch only supports haptics on the right hand. See https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#_oculus_touch_controller_profile